src/random_number.py

```diff
@@ -0,0 +1,9 @@
+#   Write a program that generates a random number.
+
+#   Output:
+#   The random number is: 4
+
+import random
+
+x = random.randint(0,1000)
```

Comment:
potresti creare una funzione che prende in input l'intervallo di valori

Reply:
Fatto